common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -42,14 +42,28 @@
 /**
  * <p>
  * Configuration of Tachyon. It sets various Tachyon parameters as key-value pairs. This class
- * contains all the runtime configuration properties. The default properties is stored in file
- * <code>tachyon-default.properties</code> and users can override these default properties by
- * modifying file <code>tachyon-site.properties</code>.
- * </p>
+ * contains all the runtime configuration properties.
+ *
+ * <p>
+ * Here is the order for this class to decide where to load the properties:
+ * <ol>
+ * <li>System properties</li>
+ * <li>Environment variables via <code>tachyon-env.sh</code> or from OS settings</li>
+ * <li>Site specific properties <code>tachyon-site.properties</code> file</li>
+ * <li>Default properties via <code>tachyon-default.properties</code> file</li>
+ * </ol>
+ *
+ * <p>
+ * The default properties are defined in a property file <code>tachyon-default.properties</code>
+ * distributed with Tachyon jar. Tachyon users can override these default properties by creating
+ * <code>tachyon-site.properties</code> and put it under java CLASSPATH when running Tachyon (e.g.,
+ * tachyon/conf/)
+ *
+ * <p>
+ * A developer can create an instance of this class by <code>new TachyonConf()</code>, which will
+ * load values from any Java system properties set as well.
```

Comment:
Are java system properties different from System properties? It may confuse a user since system properties is used before.